src/odemis/cli/main.py

```diff
@@ -24,21 +24,22 @@
 
 from __future__ import division, print_function
 
-from past.builtins import basestring, unicode
-from builtins import str
 import argparse
+from builtins import str
```

Comment:
Do we still need the builtins/past.builtins imports?

Reply:
Not really, but to remove them, we need to update the code to Python 3-only, which I'm not going to do in this PR.
I'll put the lines back to where they were.